src/pinns_pde_solve.jl

```diff
@@ -638,6 +638,22 @@ function symbolic_discretize(pde_system::PDESystem, discretization::PhysicsInfor
                                                                bc_indvars = bc_indvar) for (bc,bc_indvar) in zip(bcs,bc_indvars)]
     symbolic_pde_loss_function,symbolic_bc_loss_functions
 end
+
+function remake(pde_system::PDESystem, discretization::PhysicsInformedNN, init_params)
```

Comment:
remake always does changes via keyword arguments matching each thing. 

https://diffeq.sciml.ai/stable/basics/problem/#Modification-of-problem-types

It's done automatically through this macro:

https://github.com/SciML/SciMLBase.jl/blob/master/src/problems/ode_problems.jl#L30